.gitignore

```diff
@@ -1,6 +1,8 @@
 ABOUT-NLS
 aclocal.m4
 autom4te.cache/
+bindings/vala/*.c
+bus/ibus-daemon.1
```

Comment:
I think man pages should be managed in Git.

Reply:
???

They are currently not managed in git. They are generated by `configure`, per commit 03a486c1ec98647e7f8c7dbb70e6e4d3898c9322.

The current status is that after `./configure` is run, `git status` shows the man pages as untracked files. I think most people would consider this... not good (and the build modifying files that are checked into git by updating their copyright dates is worse—now you cannot use `git commit -a`!)

In any case, it's not really my objective to solve this problem here, so I've removed the entries from `.gitignore`.

Reply:
After just running `./configure`:

````
mattst88@framework ibus % git status
On branch build-fixes
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	deleted:    engine/simple.xml.in
	deleted:    src/compose/sequences-big-endian
	deleted:    src/compose/sequences-little-endian
	deleted:    src/ibusemojigen.h
	deleted:    src/ibusunicodegen.h

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	bus/ibus-daemon.1
	data/dconf/00-upstream-settings.5
	data/dconf/ibus.5
	pkgconfig/
	setup/ibus-setup.1
	tools/ibus.1
	ui/gtk3/ibus-emoji.7
````

Reply:
Sorry, I forgot ibus/bus/ibus-daemon.1.in file and your changes was fine.

Reply:
Restored.